benchmarks/recv-into-client.py

```diff
@@ -153,7 +153,8 @@ def serve(port, n_bytes, n_iter, recv, np, verbose, increment):
     expected = np.ones(n_bytes, dtype="u1")
     #            0 or n_iter
     expected *= int(increment) * n_iter
-    np.testing.assert_array_equal(msg, expected)
+    # breakpoint()
+    # np.testing.assert_array_equal(msg, expected)
```

Comment:
Well, just as an example, I'm guessing code like this was left over from earlier debugging. Should it be dropped? Or is there something I'm missing here?

Reply:
Ah!  Yes, I should clean that up!

Reply:
There are a few other things like this that might be worth checking on real quick. If some of them need to be there, happy to revisit.

Reply:
cleaned up.  Thanks for checking this